addons/stock_account/models/product.py

```diff
@@ -52,7 +52,7 @@ class ProductProduct(models.Model):
 
     stock_value_currency_id = fields.Many2one('res.currency', compute='_compute_stock_value_currency')
     stock_value = fields.Float(
-        'Value', compute='_compute_stock_value')
+        'Stock Value', compute='_compute_stock_value')
```

Comment:
I asked @lap-odoo about this. You should rename it to 'Total Value' to keep a correct meaning